cibuildwheel/windows.py

```diff
@@ -34,6 +35,7 @@
 
 def get_nuget_args(version: str, arch: str, output_directory: Path) -> List[str]:
     platform_suffix = {"32": "x86", "64": "", "ARM64": "arm64"}
+    platform_suffix["AMD64"] = platform_suffix["64"]  # aliased name
```

Comment:
Also need to alias the 32-bit OS here

Reply:
Or if there's already a convenient mapping from `platform.machine()` to `32`/`64`/`ARM64` that I missed, I could use that in the caller instead. But I didn't find anything